tests/unit/test_req_install.py

```diff
@@ -108,3 +112,52 @@ def test_install_req_from_string_with_comes_from_without_link(self):
         assert install_req.link.url == wheel_url
         assert install_req.req.url == wheel_url
         assert install_req.is_wheel
+
+
+class TestInstallRequirementWheelCache(object):
+
+    def test_able_to_register_when_get_from_wheel_cache(self, tmpdir):
+        """
+        This test to make sure that file is able to tell when a link is
+        from the wheel cache.
+        """
+
```

Comment:
What's `pip` policy toward PEP 257?  I see this style of docstring all over functional tests?  Is it time to start complying with the PEP?